app/v1/controllers/RetrievePeriodSummaryController.scala

```diff
@@ -46,14 +46,19 @@ class RetrievePeriodSummaryController @Inject() (val authService: EnrolmentsAuth
   implicit val endpointLogContext: EndpointLogContext =
     EndpointLogContext(controllerName = "RetrievePeriodSummaryController", endpointName = "retrieveSelfEmploymentPeriodicSummary")
 
-  def handleRequest(nino: String, businessId: String, periodId: String): Action[AnyContent] =
+  def handleRequest(nino: String, businessId: String, periodId: String, taxYear: Option[String]): Action[AnyContent] =
     authorisedAction(nino).async { implicit request =>
       implicit val correlationId: String = idGenerator.getCorrelationId
       logger.info(
         message = s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] " +
           s"with correlationId : $correlationId")
-      val rawData = RetrievePeriodSummaryRawData(nino, businessId, periodId)
-      val result =
+      val tysTaxYear: Option[String] = taxYear match {
```

Comment:
Do we need this pattern matching here? Could we not just delete this and replace

val rawData = RetrievePeriodSummaryRawData(nino, businessId, periodId, tysTaxYear)

with

val rawData = RetrievePeriodSummaryRawData(nino, businessId, periodId, taxYear) ?